docker-compose.yml

```diff
@@ -24,4 +24,20 @@ services:
         - 5000:5000
     volumes:
         - ./todo-app:/app/todo_app 
-    privileged: true
\ No newline at end of file
+    privileged: true
+
+
+volumes:
+  mongodb:
+
+services:
+  mongo:
+    image: mongo
+    volumes:
+     - mongodb:/data/db
+
+  client:
+    image: mongo
+    depends_on: [mongo]
+    entrypoint: mongo
+    command: mongodb://mongo
```

Comment:
Is there any reason for adding a client? It doesn't look like this is doing anything